duet/views/ad_activation_performance_table.view.lkml

```diff
@@ -0,0 +1,143 @@
+include: "//looker-hub/firefox_ios/views/ad_activation_performance_table.view.lkml"
+
+view: +ad_activation_performance_table {
+
+  dimension: campaign_spend {
+    hidden: yes
+    sql: ${TABLE}.campaign_spend ;;
+  }
+
+  measure: campaign_spend_sum {
+    label: "Spend"
+    type: sum
+    sql: ${TABLE}.campaign_spend ;;
+  }
+
+  dimension: clients_activated {
+    hidden: yes
+    sql: ${TABLE}.clients_activated ;;
+  }
+
+  measure: clients_activated_sum {
+    label: "Clients Activated"
+    type: sum
+    sql: ${TABLE}.clients_activated ;;
+  }
+
+  dimension: campaign_spend_per_activation {
+    hidden: yes
+    sql: ${TABLE}.campaign_spend_per_activation ;;
+  }
+
+  measure: campaign_spend_per_activation_ratio {
+    label: "Cost per Activation"
+    type: number
+    value_format_name: usd
+    sql: ${campaign_spend_sum}/ NULLIF(${clients_activated_sum},0) ;;
+  }
+
+  dimension: impressions {
+    hidden: yes
+    sql: ${TABLE}.impressions ;;
+  }
+
+  measure: impressions_sum {
+    label: "Impressions"
+    type: sum
+    sql: ${TABLE}.impressions ;;
+  }
+
+  dimension: new_downloads {
+    hidden: yes
+    sql: ${TABLE}.new_downloads ;;
+  }
+
+  measure: new_downloads_sum {
+    label: "New Downloads"
+    group_label: "Downloads"
+    type: sum
+    sql: ${TABLE}.new_downloads ;;
+  }
+
+  dimension: redownloads {
+    hidden: yes
+    sql: ${TABLE}.redownloads ;;
+  }
+
+  measure: redownloads_sum {
+    label: "Redownloads"
+    group_label: "Downloads"
+    type: sum
+    sql: ${TABLE}.redownloads ;;
```

Comment:
```suggestion
    sql: ${TABLE}.redownloads ;;
    description: "Redownloads occur when a user downloads your app, deletes it, and downloads the same app again following a tap on an ad on the App Store, or downloads the same app on an additional device."
```

Reply:
These came from https://searchads.apple.com/help/reporting/0023-reporting-options-and-definitions